usaspending_api/database_scripts/matviews/vw_search_award.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN;
```

Comment:
Given there's no data being generated with this (like with a matview), it seems like it would make more sense to create this view as part of a Django migration. If alterations are needed, we would write another migration to alter the view.

Reply:
Unfortunately this cannot be managed by Django since it depends on non-django managed materialized views. Tests fail since this migration would always fail

Reply:
I see... has to be run _after_ the matview schema objects are created. 
Takes us back to `USAspending DB Initialize and Update` Epic: https://federal-spending-transparency.atlassian.net/browse/DEV-982